src/functions/moveChecks/bishopCheck.ts

```diff
@@ -8,45 +8,70 @@ import {
 	lastRow,
 	firstRow,
 	diagonalRowSE,
-	diagonalRowNW
+	diagonalRowNW,
+	hasOwnPiece,
+	hasOpponentPiece
 } from '../../global';
+import type { Square } from '../../stores';
 
-export const bishopCheck = (targetSquare: number) => {
+export const bishopCheck = (targetSquare: number, board: Square[], turn: string) => {
 	const tempArray: number[] = [];
 	const startSquare = targetSquare;
```

Comment:
Why are you renaming this here? If you want the name to be startSquare, you can just name it that in the arguments

Reply:
resolved